components/dashboard/src/Analytics.tsx

```diff
@@ -40,6 +41,8 @@ export interface TrackFeedback {
     feedback: string;
     href: string;
     path: string;
+    error_object?: StartWorkspaceError;
+    error_message?: string;
```

Comment:
Is there a reason these props have to be snake-cased rather than camel-cased?

Reply:
Not really! Before it was `error_rendered` which I confirmed with @jakobhero. Let me see what the convention is in Segment.

Reply:
I see it's a mix 😅 Some are `ui_experiments`, `button_type`, while others are `workspaceInstanceId ` or `clientKind`.

Reply:
Asked [here](https://gitpod.slack.com/archives/C01KPEPKX0A/p1663243627662879)

Reply:
Sorry, I thought this interface defined props for some component, but it doesn't 😊 It's fine how it is if Segment needs it that way.